### PR TITLE
update go.mod

### PR DESCRIPTION
**Pull request by XuHuaiyu**
update go.mod for https://github.com/pingcap/parser/pull/232

### CODE REVIEW

**Comment by zz-jason** [comment]
LGTM

**Comment by zz-jason** [comment]
/run-all-tests tidb-test=release-2.1 pd=release-2.1 tikv=release-2.1

File: go.sum [hunk 1]
```diff
@@ -90,14 +90,10 @@ github.com/pingcap/gofail v0.0.0-20181217135706-6a951c1e42c3 h1:04yuCf5NMvLU8rB2
 github.com/pingcap/gofail v0.0.0-20181217135706-6a951c1e42c3/go.mod h1:DazNTg0PTldtpsQiT9I5tVJwV1onHMKBBgXzmJUlMns=
 github.com/pingcap/goleveldb v0.0.0-20171020084629-8d44bfdf1030 h1:XJLuW0lsP7vAtQ2iPjZwvXZ14m5urp9No+Qr06ZZcTo=
 github.com/pingcap/goleveldb v0.0.0-20171020084629-8d44bfdf1030/go.mod h1:O17XtbryoCJhkKGbT62+L2OlrniwqiGLSqrmdHCMzZw=
-github.com/pingcap/kvproto v0.0.0-20190225084405-84f2c621d8e8 h1:ZoP49RWRjlmvXUWAySYZD1tV8BIVVEJ7xrbCg1B7/fw=
```
Comment (eurekaka) [inline go.sum]: Is this change related?

**Comment by eurekaka** [comment]
/run-all-tests tidb-test=release-2.1 pd=release-2.1 tikv=release-2.1

File: go.sum (hunk 1 above)
Comment (XuHuaiyu) [reply go.sum]: No, I run `go mod tidy`, and this happens.

**Comment by XuHuaiyu** [comment]
/run-all-tests tidb-test=pr/750 pd=release-2.1 tikv=release-2.1

**Comment by XuHuaiyu** [comment]
/run-all-tests tidb-test=pr/750 pd=release-2.1 tikv=release-2.1
